user_tools/src/spark_rapids_pytools/common/utilities.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2023, NVIDIA CORPORATION.
+# Copyright (c) 2023-2024, NVIDIA CORPORATION.
```

Comment:
Looks like you updated this file's copyright instead of `data_validation/src/spark_rapids_validation_tool/utilities.py`

Reply:
sorry about this, now should be fixed.